src/wasm.rs

```diff
@@ -11,16 +11,13 @@ pub use self::lang::Flux;
 pub use self::lsp::Lsp;
 
 use flux::{ast, formatter, parser};
+use log::Level;
 use wasm_bindgen::prelude::*;
+use wasm_logger;
 
-/// Initialize logging - this requires the "console_log" feature to function,
-/// as this library adds 180k to the wasm binary being shipped.
-#[allow(non_snake_case, dead_code)]
-#[wasm_bindgen]
-pub fn initLog() {
-    #[cfg(feature = "console_log")]
-    console_log::init_with_level(log::Level::Trace)
```

Comment:
Chose wasm-logger over console-log lib, due to the [reported build size](https://crates.io/crates/console_log#user-content-code-size).